descriptors/ModuleDescriptor-template.json

```diff
@@ -343,14 +357,40 @@
       "displayName" : "Get refreshed presigned url for export file",
       "description" : "Get refreshed presigned url for export file"
     },
+    {
+      "permissionName" : "permissions-self-check.get",
```

Comment:
give more specific name to avoid duplicates across modules, better to follow naming convention

Reply:
The permission and endpoint were updated to avoid duplications while execution selfcheck endpoint.